File: python_scripts/persist_load_test.py
import emu
import savestate

print("PY: Start persist_load_test.py")

while True:
    if emu.framecount() == 5 * 60:
        ss = savestate.object(1)
        print("PY: LOAD", ss)
        savestate.load(ss)
    
    emu.frameadvance()

File: python_scripts/persist_load_test.lua
print("Start persist_load_test.py")

while (1) do
    if (emu.framecount() == 5 * 60) then
        ss = savestate.object(2)
        print("Load", ss)
        savestate.load(ss)
    end

    emu.frameadvance()
end
